examples/kitchen-sink/apps/blog/remix.config.js

```diff
@@ -1,9 +1,16 @@
 /**
- * @type {import('@remix-run/dev/config').AppConfig}
+ * @type {import('@remix-run/dev').AppConfig}
  */
 module.exports = {
-  appDirectory: "app",
-  browserBuildDirectory: "public/build",
-  publicPath: "/build/",
-  serverBuildDirectory: "api/build",
+  serverBuildTarget: "vercel",
+  // When running locally in development mode, we use the built in remix
+  // server. This does not understand the vercel lambda module format,
+  // so we default back to the standard build output.
+  server: process.env.NODE_ENV === "development" ? undefined : "./server.js",
+  ignoredRouteFiles: [".*"],
+  // appDirectory: "app",
+  // assetsBuildDirectory: "public/build",
+  // serverBuildPath: "api/index.js",
+  // publicPath: "/build/",
+  serverDependenciesToBundle: ["ui"],
```

Comment:
This tells Remix to rebuild and refresh the page.
FYI: This only works based on the [patch](https://github.com/ruisaraiva19/turborepo/blob/ruisaraiva/upgrade-remix/examples/kitchen-sink/patches/@remix-run+dev+1.4.3.patch) done to the `@remix-run/dev/compiler.js` file.